frontend/scripts/check-patternfly-modules.sh

```diff
@@ -68,9 +71,18 @@ PKGS_TO_CHECK=(
 
 echo -e "Checking ${COL_YELLOW}yarn.lock${COL_RESET} file for PatternFly module resolutions"
 
-for pkg in "${PKGS_TO_CHECK[@]}"; do
-  check-resolution "${pkg%%:*}" "${pkg#*:}"
-done
+if [ "$OPENSHIFT_CI" = true ]; then
```

Comment:
Nit - I'd suggest to stick with Bash specific syntax, since this is a `#!/bin/bash` script.

https://devhints.io/bash

```suggestion
if [[ "$OPENSHIFT_CI" == "true" ]]; then
```